tensorflow/core/common_runtime/pluggable_device/pluggable_device_plugin_init.cc

```diff
@@ -81,13 +82,26 @@ static Status InitKernelModule(void* dso_handle) {
   return Status::OK();
 }
 
+static Status InitProfilerModule(void* dso_handle) {
+  void* dso_symbol;
+  tensorflow::Env* env = tensorflow::Env::Default();
+
+  TF_RETURN_IF_ERROR(
+      env->GetSymbolFromLibrary(dso_handle, "TF_InitProfiler", &dso_symbol));
+  auto init_fn = reinterpret_cast<profiler::TFInitProfilerFn>(dso_symbol);
+  TF_RETURN_IF_ERROR(profiler::InitPluginProfiler(init_fn));
+  return Status::OK();
+}
+
 Status RegisterPluggableDevicePlugin(void* dso_handle) {
   // Step 1 Init Device/Graph Module
   TF_RETURN_IF_ERROR(InitDeviceAndGraphModule(dso_handle));
 
   // Step 2 Init Kernel Module
   TF_RETURN_IF_ERROR(InitKernelModule(dso_handle));
 
+  // Step 3 Init Profiler Module(optional module)
+  InitProfilerModule(dso_handle);
```

Comment:
A returned Status must not be ignored. If not using TF_RETURN_IF_ERROR, explicity call IgnoreError() on the result of the call and explain in a comment why it is OK to ignore. Even better, log the resulting status so for ease of debugging.

Reply:
Done